rust/src/nfs/nfs2_records.rs

```diff
@@ -96,3 +96,211 @@ pub fn parse_nfs2_attribs(i: &[u8]) -> IResult<&[u8], Nfs2Attributes> {
     let attrs = Nfs2Attributes { atype, asize };
     Ok((i, attrs))
 }
+
+#[cfg(test)]
+mod tests {
+    use crate::nfs::nfs2_records::*;
+    use nom7::bytes::streaming::take;
+    use nom7::IResult;
+
+    #[test]
+    fn test_nfs2_handle() {
+
+        // file_handle bytes
+        let buf: &[u8] = &[
+            0x00, 0x10, 0x10, 0x85, 0x00, 0x00, 0x03, 0xe7,
+            0x00, 0x0a, 0x00, 0x00, 0x00, 0x00, 0xb2, 0x5a,
+            0x00, 0x00, 0x00, 0x29, 0x00, 0x0a, 0x00, 0x00,
+            0x00, 0x00, 0xb2, 0x5a, 0x00, 0x00, 0x00, 0x29
+        ];
+
+        let result = parse_nfs2_handle(buf);
+        match result {
+            Ok((r, res)) => {
+                assert_eq!(r.len(), 0);
+                assert_eq!(res.value, buf);
+            }
+            Err(error) => { panic!("Parsing nfs2_handle failed: {:?}", error); }
+        }
```

Comment:
Constructs like these could be an assert statement prior to other asserts. Something like:
`assert_eq!(true, result.is_ok())`

Reply:
I'll consider this with the other PR too, thanks!